modules/core/src/main/java/org/apache/ignite/mxbean/ThreadPoolMXBean.java

```diff
@@ -143,7 +143,7 @@
      *
      * @return Class name of current rejection handler.
      */
-    @MXBeanDescription("Class name of current rejection handler.")
+    @MXBeanDescription("Handler class name for rejected tasks.")
```

Comment:
Maybe, it would be good to copy&paste javadoc for See `java.util.concurrent.RejectedExecutionHandler`,
i.e. `The class name of a handler for tasks that cannot be executed by a thread pool.`
What do you think?

Reply:
Please check